### DISCUSSION TITLE
Weekly Meeting 2nd Aug 2022

### DISCUSSION THREAD

**Discussion by AdamFrisby** [omigroup/omi-scripting-group#36] [General]
Topics:
- Discussion of time was considered. Was proposed to move to 1700 UTC on Thursdays as this works better for some people; **resolution: leave this up for a week here, see if there are any comments/feedback before next weeks call (still at 1600 UTC Tuesday)**
- Discussion of Behavior Graph from Ben Houston: suggestion to see if he wants to present at an upcoming meeting.
- Suggestion of splitting the Developer Hour (/ Platform Implementors) from the scripting, running each on a bi-weekly cadence.
- Discussion of getting a "bare bones" API, Robert/Adam discussed ripping one out of the sinespace API (Adam volunteered to help if possible) and creating a new minimal "cross-platform" API. Discussion to be resumed next week.

I'm missing notes from the first 30 minutes, if people could backfill those, that would be great.

**Comment by robertlong** [comment]
I think moving the meetings to 1700 UTC on Thursdays is great and also works well for me even with daylight savings time occurs.

I left more comments on the developer hour split here: https://github.com/omigroup/omi-scripting-group/issues/31
